packages/babel-preset-stage-2/src/index.js

```diff
@@ -37,7 +37,7 @@ export default declare((api, opts = {}) => {
   return {
     presets: [[presetStage3, { loose, useBuiltIns }]],
     plugins: [
-      [transformDecorators, { legacy: decoratorsLegacy }],
+      [transformDecorators, { legacy: decoratorsLegacy && false }],
```

Comment:
this will prevent `decoratorsLegacy` from ever affecting the value. It will always be false?

Reply:
Nope, it's just to test decorators in the REPL before that this PR is meged. That is why I labeled the commit as `[REVERT BEFORE MERGING]` :stuck_out_tongue:

Reply:
oh, Ha... I didn't see the commit message :) sorry.